tests/unit/metadata/test_multi_table.py

```diff
@@ -1195,21 +1195,16 @@ def test_validate_raises_errors(self):
 
         # Run
         error_msg = re.escape(
-            'The metadata is not valid\n'
-            '\nTable: payments'
-            "\nInvalid regex format string '[A-z{' for id column 'date'."
-            '\n\nRelationships:'
-            "\nThe parent table 'users' does not have a primary key set. "
-            "Please use 'set_primary_key' in order to set one."
-            "\nRelationship between tables ('sessions', 'transactions') is invalid. "
-            'The primary and foreign key columns are not the same type.'
-            "\nThe relationships in the dataset are disjointed. Table ['payments'] "
-            'is not connected to any of the other tables.'
+            'The relationships in the dataset are disjointed. '
+            "Table ['payments'] is not connected to any of the other tables."
         )
 
         # Run and Assert
         with pytest.raises(InvalidMetadataError, match=error_msg):
-            instance.validate()
+            instance._validate_all_tables_connected(
```

Comment:
Can we have this be an entirely new unit test instead of replacing `test_validate_raises_errors`?

Reply:
Yeah I was thinking we change this to be a different unit test and changing the name. As for the pre-existing test, not sure if that is valid anymore since we won't throw that error anymore.

Reply:
Shouldn't the original test still throw the error just without the disconnected schema entry?

Reply:
Oh yes the other issues, sorry I forgot all the errors got appended into a single message